src/appshell/qml/platform/PlatformMenuBar.qml

```diff
@@ -104,6 +100,8 @@ Item {
             property var subitems: []
 
             onAboutToShow: {
+                for(var j in items)
+                    items[j].destroy()
```

Comment:
run: error: qrc:/qml/platform/PlatformMenuBar.qml:106: Error: Invalid attempt to destroy() an indestructible object

macOS

Reply:
That's strange, it fixes the crash on Linux for some reason. I'll have a look at the logs...